### PR TITLE
Adds shrimp carbons

### PR DESCRIPTION
**Pull request by vampirebat74**
for the funny

## About The Pull Request

Adds a new carbon to the game. **It cannot be selected at the start of the round,** only appearing in-game through wishing well without admin intervention, but ghosts are able to possess them easily.

## Why It's Good For The Game

Administrators and players seemed to have a lot of fun with Pinnochio. Even with all of the restrictions in place, carbons generally offer a lot in terms of interaction. I figured this would forward the main objective this game serves: to be fun.

## Changelog
:cl:
add: Adds a new playable carbon
imageadd: adds relevant body part graphics
/:cl:

### CODE REVIEW

**Comment by vampirebat74** [comment]
Todo: Add name selection on possession

File: code/modules/mob/living/carbon/human/species_types/shrimp.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/datum/species/shrimp
+	name = "Shrimp"
+	id = "shrimp"
+	mutant_bodyparts = list()
+	say_mod = "burbles"
+	sexes = 0
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/carbon/human/species_types/shrimp.dm:6]: Shrimp has no bitches.

**Review by Kitsunemitsu — DISMISSED** [review]
Code looks to be in order; yeah sure let's merge it.

**Review by Kitsunemitsu — APPROVED** [review]
Just added in names since the last time I reviewed

**Pull request merged**